components/blocks/FilesToCards.vue

```diff
@@ -284,6 +279,23 @@ export default {
     clearAll() {
       this.searchGroup = [];
     },
+    contributors(card) {
+      const subset = (({investigators, people}) => ({investigators, people}))(card)
```

Comment:
this is a cool trick! To be consistent with how we handle tags, I might instead add to the data an contributors object with `{ investigators: 'mdi-brain', people: 'mdi-account-multiple' } then use that similar to the tags in the markup

Reply:
`contributors()` is accessing the investigators (if they exist) and people tags from the data object for each card - not the icons.